easybuild/easyconfigs/t/TAU/TAU-2.22.2-gompi-1.4.12-no-OFED.eb

```diff
@@ -0,0 +1,34 @@
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+# Copyright:: Copyright 2013 Juelich Supercomputing Centre, Germany
+# Authors::   Bernd Mohr <b.mohr@fz-juelich.de>
+# License::   New BSD
+#
+# This work is based from experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+name = 'TAU'
+version = '2.22.2'
+
+homepage = 'http://tau.uoregon.edu'
+description = """The TAU Performance System is a portable profiling and tracing toolkit
+ for performance analysis of parallel programs written in Fortran, C, C++, Java, Python."""
+
+toolchain = {'name': 'gompi', 'version': '1.4.12-no-OFED'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://www.cs.uoregon.edu/research/paracomp/tau/tauprofile/dist/']
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('OTF', '1.12.4'),
+    ('PAPI', '5.2.0'),
+    ('PDT', '3.19'),
+    ('Scalasca', '1.4.3'),
+    ('Score-P', '1.2.1'),
```

Comment:
@geimer I just tried both options, and both failed...

* it seems like Score-P really wants Cube 4.x, otherwise it complains about a particular options for one of the Cube tools
* using Cube 4.2 for Scalasca requires that I bump OPARI2 to v1.1.1 (from v1.0.7), but then I run into this when building Scalasca 1.4.3

```
epk_pomp2_base.c:119:1: error: conflicting types for ‘POMP2_Begin’ 
```

Any ideas about a compatible combo of Scalasca 1.x and Score-P that can be used as deps for TAU?